directory/cli/src/user.py

```diff
@@ -74,7 +74,93 @@ def show(login):
             error = '{}: user not found'.format(login)
             raise click.ClickException(error)
 
-    wrapper_commands = [
+    if not utils.advanced_mode_enabled():
+        @user.command(help='Create a new user')
+        def create():
+            click.echo('Please enter details for the following:')
+
+            params = {
+                        'login': click.prompt('  Username'),
+                        'first': click.prompt('  First name'),
+                        'last': click.prompt('  Surname'),
+                        'email': click.prompt('  Email'),
+            }
+            if click.confirm('Would you like to add a UID to this user?'):
+                params = { **params, 'uid': click.prompt('  UID') }
+
+            group_id = _get_group_id('clusterusers')
+
+            if group_id:
```

Comment:
Do we/why do we need this conditional? Won't `_get_group_id` always give the `group_id`, and if the group can't be found raise a `ClickException` which will be caught and displayed at the top level (and so this conditional will never be hit with a falsy `group_id`)?

Reply:
Truthfully this was mostly added to stop tests complaining about `NoneType` objects not being subscriptable. Arguably not the best way to handle this and I am open to suggestions.

Reply:
I believe we decided this was able to be removed following the improvements we made to how we're testing this stuff yesterday? Or is it still needed for some reason?